Pipfile

```diff
@@ -15,6 +15,7 @@ sparkmonitor = "==3.0.2"
 itables = "==2.1.1"
 PyMySQL = "==1.1.1"
 urllib3 = "==2.2.2"
+jupyterlab-git = "==0.50.1"
```

Comment:
Do docs need to be added to explain to users how to make use of this?

Reply:
I think it's pretty self-explanatory as the extension added an UI component. Users don't need to do any extra steps to enable this. 

<img width="372" alt="Screenshot 2024-06-24 at 11 40 56 AM" src="https://github.com/kbase/cdm-jupyterhub/assets/3953823/c8f830d6-95a2-4141-968e-45f8394dbc7f">

Reply:
:+1: